reference-architecture/azure-ansible/azuredeploy.json

```diff
@@ -286,23 +286,23 @@
       }
     },
     "metrics" : {
-      "type": "bool",
-      "defaultValue": true,
-      "metadata" :{
+      "type" : "bool",
+      "defaultValue" : true,
+      "metadata" : {
         "description" : "Enable OCP metrics"
       }
     },
     "logging" : {
-      "type": "bool",
-      "defaultValue": true,
-      "metadata" :{
+      "type" : "bool",
```

Comment:
Actually alot of these are triggered by comma changes and spaces.
The comma changes are mandatory, the spaces changes are caused by a tool chain
difference. I will suggest to edu to use the same tool for editing, and we can avoid 
some of the white space. But things its also lack of smarts on git for json format, as its
not a material change.